packages/docusaurus-module-type-aliases/src/index.d.ts

```diff
@@ -53,6 +53,28 @@ declare module '@theme-original/*';
 
 declare module '@docusaurus/*';
```

Comment:
This globally applies an `any` to all `@docusaurus/*` stuff, but because the following are more specific, then they get picked up as the module definition. This means as you add new things to the module TS users just get any, but then it can get filled in here.

Reply:
![Screen Shot 2020-08-08 at 11 26 02 AM](https://user-images.githubusercontent.com/49038/89714155-be425380-d96a-11ea-9b14-7eec10eb1f29.png)